templates/admin/console.html

```diff
@@ -0,0 +1,27 @@
+{% extends "layout.html" %}
+
+{% block title %}Admin Console{% endblock %}
+
+{% block main %}
+<h1>Admin Console</h1>
+<a class="btn btn-primary" href="/admin/submissions" role="button">Submissions</a>
+<br>
+<br>
+<a class="btn btn-primary" href="/admin/users" role="button">Users</a>
+<br>
+<br>
+<a class="btn btn-primary" href="/admin/createannouncement" role="button">Create Announcement</a>
+<br>
+<br>
+<a class="btn btn-primary" href="/admin/createcontest" role="button">Create Contest</a>
+<br>
+<br>
+<a class="btn btn-primary" href="/admin/createproblem" role="button">Create Problem</a>
+<br>
+<br>
+<a class="btn btn-primary" href="/problems/draft" role="button">Draft Problems</a>
+<hr>
+<p>To disable maintenance mode, visit the /admin/maintenance webpage. If you log out during maintenance mode, you can only turn off maintenance mode with a full application reboot.</p>
```

Comment:
This is not true anymore, admins can log in and then regain access again

Reply:
Also, remove this `To disable maintenance mode, visit the /admin/maintenance webpage.`